documentation/asciidoc/microcontrollers/debug-probe/swd-connection.adoc

```diff
@@ -35,10 +37,13 @@ $ make -j4
 ----
 
 In a debug build you will get more information when you run it under the debugger, as the compiler builds your program with the information to tell GDB what your program is doing.
-
-See Chapter 6 of https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf[Getting started with Raspberry Pi Pico] for more information.
 ======
 
+NOTE: For devices that are _not_ Raspberry Pis, a variant of GDB that can debug ARM processors is required. Use one of the following alternatives depending on your operating system and device:
```

Comment:
"For devices that are _not_ Raspberry Pis" - might be worth changing "devices" to "computers", in order to avoid possible confusion with Raspberry Pi Pico devices?